docs/Developers/Writing_Good_Commits.rst

```diff
@@ -115,7 +115,7 @@ Example:
 
 ::
 
-    Fixes https://github.com/coala/coala/issues/269
+    Closes https://github.com/coala/coala/issues/269
```

Comment:
Sure, we can consider the Quick Reference example and then split it into shortlog, message body and issue reference.